$NetBSD: patch-ae,v 1.2 2008/03/18 11:48:25 dmcmahill Exp $

- Rename tnt to tnt-mmtl to avoid conflicts with another pacakge.  

- Put the examples and docs in the right place

- Do not install calcCAP or calcRL.  These two tools seem to give
results of questionable accuracy (see the sourceforge bug tracker
for the upstream sources) and also don't build with gcc-4.

--- Makefile.am.orig	2004-07-22 14:23:32.000000000 +0000
+++ Makefile.am	2008-03-17 13:19:02.000000000 +0000
@@ -28,10 +28,10 @@
 #  tcl/tk scripts that need to be installed.
 #---------------------------------------------------------------
-bin_SCRIPTS = tnt.tcl tnt
+bin_SCRIPTS = tnt.tcl tnt-mmtl
 
 #---------------------------------------------------------------
 #  Examples to be installed
 #---------------------------------------------------------------
-examplesdir = $(prefix)/examples
+examplesdir = $(pkgdatadir)/examples
 examples_DATA = \
 	examples/9-7-00.xsctn \
@@ -52,5 +52,5 @@
 #  /usr.
 #---------------------------------------------------------------
-appdocsdir = $(prefix)
+appdocsdir = $(pkgdatadir)
 appdocs_DATA = \
 	AUTHORS$(TXTEXT) \
@@ -84,5 +84,6 @@
 #  Subdirectories for automake and make
 #---------------------------------------------------------------
-SUBDIRS = bem calcCAP calcRL doc csdl gui units sppdgTcllib
+#SUBDIRS = bem calcCAP calcRL doc csdl gui units sppdgTcllib
+SUBDIRS = bem doc csdl gui units sppdgTcllib
 
 EXTRA_DIST = $(bin_SCRIPTS) $(printfile_DATA) $(examples_DATA)
